.codeclimate.yml

```diff
@@ -0,0 +1,11 @@
+engines:
+ gofmt:
+   enabled: true
+ golint:
+   enabled: true
+ govet:
+   enabled: true
+ratings:
+ paths:
+ - "**.go"
+exclude_paths:
```

Comment:
At least the following directories and files should be excluded (directories recursively):
- ./vendor
- ./web/ui/static/vendor
- ./web/ui/bindata.go